### ISSUE TITLE
[Bug] When adding an item with relationships

### ISSUE THREAD

**Issue by Alerinos** [KorzhCom/EasyData#114]
After adding an item to a model that has a relation, freez takes place. The solution is a page refresh.
error:
```
easydata.min.js:2 
        
       POST http://localhost:5021/crud/articles/models/__default/sources/Article/create 400 (Bad Request)
```
Video:
https://gyazo.com/bd9c74000ee1cac8fde54d3f8f3fbeb9

Models:
[ArticlesContext.zip](https://github.com/KorzhCom/EasyData/files/7943881/ArticlesContext.zip)

Main model:
```cs
public class Article
{
    public Guid Id { get; set; }
    public StatusType Status { get; set; } = StatusType.Draft;
    public int Views { get; set; } = 0;                         // Wyświetlenia
    public string Culture { get; set; } = "en-US";              // Język wyświetlania

    public ICollection<Names>? Names { get; set; }                           // Lista nazw artykułu (Jeżeli zmienisz nazwę to generuje się nowy link, stary też musi działać)
    public Details Details { get; set; } = new Details();       // Szczegóły artykułu
    public ICollection<HashTag>? HashTags { get; set; }         // Lista tagów, np popular, hot, mmo itp
    public ICollection<Image>? Images { get; set; }             // Lista obrazków w artykule

    public enum StatusType
    {
        Hide,
        Show,
        Unpublish,                      // Wersja nie publiczna, dostępna po wysłaniu linku
        Verification,                   // Do weryfikacji przez redaktora
        Draft                           // Wersja robocza (Automatycznie zapisuje się artykuł przed zmianem statusu)
    }
}
```

**Comment by antifree** [comment]
Could you please open developers' tools when the page freeze and see what error is there at this moment?
If there is 
```
easydata.min.js:2 
        
       POST http://localhost:5021/crud/articles/models/__default/sources/Article/create 400 (Bad Request)
``` 
Could you please investigate the content of this response in the Network and share it here?

**Comment by Alerinos** [comment]
@antifree 
![image](https://user-images.githubusercontent.com/42449535/152444705-3c92db68-2c85-4d13-a7d5-7aaccc669299.png)
I just have something like that, unfortunately the JS project is on your server so it's hard to debug.

**Comment by antifree** [comment]
You can see the content of the response in the Network tab

**Comment by korzh** [comment]
This bug has been fixed in the latest update, version 1.4.3-beta01
